src/oom/test-oomd-util.c

```diff
@@ -77,6 +77,9 @@ static void test_oomd_cgroup_kill(void) {
                         abort();
                 }
 
+                assert_se(cg_get_xattr_malloc(SYSTEMD_CGROUP_CONTROLLER, cgroup, "user.oomd_ooms", &v) >= 0);
+                assert_se(memcmp(v, i == 0 ? "1" : "2", 2) == 0);
+
```

Comment:
this also needs

```c
v = mfree(v);
```

to avoid leaking memory (since the pointer is reassigned in the `cg_get_xattr_malloc()` call below).